Riot/Modules/Common/Buttons/RoundedButton.swift

```diff
@@ -22,6 +22,7 @@ final class RoundedButton: CustomRoundedButton, Themable {
     
     private enum Constants {
         static let backgroundColorAlpha: CGFloat = 0.2
+        static let titleDisabledColorAlpha: CGFloat = 0.5
```

Comment:
If this change is not necessary on Element iOS please add a Tchap comment, otherwise you should suggest this change on Element too.

Reply:
Tchap comment added.
But I think you should think about integrating it into Element code base.

Reply:
Feel free to submit your own PR in Element iOS by following the [contribution guide](https://github.com/vector-im/element-ios/blob/develop/CONTRIBUTING.md). You could add me in revewiers if you want.

Reply:
I will.
Can you approve this PR before I do the PR Element side?

Reply:
I opened an issue on Element-ios repository : https://github.com/vector-im/element-ios/issues/7395

But I can't push my proposed changes to repository. Perhaps because I try to create a branch? (ERROR: Permission to vector-im/element-ios.git denied to NicolasBuquet)